internal/cli/atlas/backup/compliancepolicy/encryptionatrest/encryptionatrest.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package encryptionatrest
+
+import (
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	store "github.com/mongodb/mongodb-atlas-cli/internal/store/atlas"
+	"github.com/spf13/cobra"
+)
+
+const (
+	active = "ACTIVE"
+)
+
+type combinedStore interface {
+	store.CompliancePolicyEncryptionAtRestUpdater
+	store.CompliancePolicyDescriber
+}
+
+func baseCommand() *cobra.Command {
```

Comment:
not sure what the goal of having this as a dedicated method is but in go methods that return a new struct then to have `new` prefix
```suggestion
func newEncryptionAtRestCommand() *cobra.Command {
```